vllm/worker/model_runner.py

```diff
@@ -214,15 +215,27 @@ def __init__(
             dtype=np.int32)
         num_attn_heads = self.model_config.get_num_attention_heads(
             self.parallel_config)
-        self.attn_backend = get_attn_backend(
-            num_attn_heads,
-            self.model_config.get_head_size(),
-            self.model_config.get_num_kv_heads(self.parallel_config),
-            self.model_config.get_sliding_window(),
-            self.model_config.dtype,
-            self.kv_cache_dtype,
-            self.block_size,
-        ) if num_attn_heads else None
+        if getattr(self.model_config.hf_config, "dual_chunk_attention_config",
```

Comment:
We should add `dual_chunk_attention` to `ModelConfig` to accessing the `hf_config` from here. We ran into problems with accessing the `hf_config` with Sliding Window in the past